quicksort/quick_sort.py

```diff
@@ -9,13 +24,14 @@ def partition(array, p, r):
     :return: return q value for function, used in partitioning of array.
```

Comment:
Modifying this would be better, this function returns the partition index which I think should be mentioned.